<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mantenimientoTitulo</class>
 <widget class="QDialog" name="mantenimientoTitulo">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Material Bibliografico</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Busqueda</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLineEdit" name="busqueda"/>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="btmTitulo">
        <property name="text">
         <string>Titulo</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="btmAutor">
        <property name="text">
         <string>Autor</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btmBuscar">
        <property name="text">
         <string>Buscar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelTitulo">
     <property name="text">
      <string>Titulos:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QTableView" name="titulos">
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="toolButton_3">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/iconos/mas.png</normaloff>:/iconos/mas.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_2">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/iconos/menos.png</normaloff>:/iconos/menos.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/iconos/default-applications-capplet.png</normaloff>:/iconos/default-applications-capplet.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="2">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Titulo Obra</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Autores:</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Lista:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QListView" name="listaAutores">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777214</height>
             </size>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Nombre:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Apellido:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="4" column="4">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="toolButton_6">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/iconos/mas.png</normaloff>:/iconos/mas.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_5">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/iconos/menos.png</normaloff>:/iconos/menos.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_4">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/iconos/default-applications-capplet.png</normaloff>:/iconos/default-applications-capplet.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="seleccionarTitulo">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/iconos/button_ok.png</normaloff>:/iconos/button_ok.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>busqueda</sender>
   <signal>returnPressed()</signal>
   <receiver>btmBuscar</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
